sciurus17_examples_py/sciurus17_examples_py/gripper_control.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2024 RT Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import math
+
+
+# moveit python library
+from moveit.core.robot_state import RobotState
+from moveit.planning import (
+    MoveItPy,
+    PlanRequestParameters,
+)
+# generic ros libraries
+import rclpy
+from rclpy.logging import get_logger
+
+from sciurus17_examples_py.utils import plan_and_execute
+
+
+def main(args=None):
+    rclpy.init(args=args)
+    logger = get_logger('moveit_py.gripper_control')
+
+    # instantiate MoveItPy instance and get planning component
+    sciurus17 = MoveItPy(node_name='gripper_control')
+    logger.info('MoveItPy instance created')
+
+    # 腕制御用 planning component
+    arm = sciurus17.get_planning_component('two_arm_group')
+    # 左グリッパ制御用 planning component
+    l_gripper = sciurus17.get_planning_component('l_gripper_group')
+    # 右グリッパ制御用 planning component
+    r_gripper = sciurus17.get_planning_component('r_gripper_group')
+
+    robot_model = sciurus17.get_robot_model()
```

Comment:
`robot_state`の作成はプランニングごとでなく、このタイミングでいいかと思います。

```suggestion
    robot_model = sciurus17.get_robot_model()
    robot_state = RobotState(robot_model)
```

Reply:
これで問題なさそうでしたら他のサンプルにも反映していただきたいです。

Reply:
そうですね。最初に持ってきても悪影響はないのですが、前半で書き換えたrobot_stateを後半でもそのまま使いまわすことになり、指定する場所以外のところにも値が入るのが少し気持ち悪いかなと思い、各位置に置いていましたが、どうでしょうか

Reply:
承知いたしました。ではそのままで問題ないです。